### DISCUSSION TITLE
ID photo for application

### DISCUSSION THREAD

**Discussion by arduinoprojectfiles** [community/community#123428] [GitHub Education] [answered]
### Hi everyone,

I'm experiencing trouble uploading my ID card photo to the application form. Each time I attempt to upload it and hit "process my application", the page reloads without any progress. Any suggestions on resolving this issue?

**Comment by Rishit-katiyar** [comment, marked as answer]
First off, double-check that your ID photo is in the right format and not too large. Also, ensure your internet connection is stable, try switching browsers, or clear out your browser's cache and cookies. If none of that works, reach out to your school's support team for extra help. On a side note, I ran into a similar issue once and found a workaround by downloading the GitHub mobile app. Uploaded my photo there, and voila! Got a nice little 'thanks for uploading' message. Worth a shot!

**Reply by Rishit-katiyar** [reply]
![329783326-1a243b07-661c-4c7e-9ae1-c6e81a903cfe](https://github.com/community/community/assets/167756997/206a9ce1-acf8-4648-b970-8d79b352b5ba)

**Comment by praveen-svg755** [comment]
If the platform forces only camera input, you might need to reach out to their support team and request a manual upload option.
